google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -46,6 +46,7 @@ appengine.cloudsdk.java.components.missing=Selected Cloud SDK does not contain t
 appengine.cloudsdk.java.components.howtoinstall=To install, run 'gcloud components install app-engine-java'.
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
+appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters (<>:"?*) or starts or ends with spaces.
```

Comment:
I actually wanted to start a dialog about the value of pointing out to the user which characters are actually considered invalid, instead of letting the user out on a witch hunt.

I'm sensitive to your point about not being sure of what can cause an IPE. By experience, we know that this is the only case when it is happening, but there might be other cases we're overlooking.

I'll go ahead and remove the special characters parenthesis.